.circleci/config.yml

```diff
@@ -94,6 +96,10 @@ workflows:
                 name: Authenticate with FontAwesome registry
                 command: 'npm config set "//npm.fontawesome.com/:_authToken" $FONTAWESOME'
       - cypress/run:
+          pre-steps:
+            - jq/install
+            - browser-tools/install-firefox
+          browser: firefox
```

Comment:
ok so it looks like firefox will work now.  Is the plan to remove this default until edge is available?

Reply:
Yes, I'm just running a quick local test to verify, then I'll remove this default.